api/setting.go

```diff
@@ -126,11 +126,16 @@ func (rs RedundancySettings) Redundancy() float64 {
 	return float64(rs.TotalShards) / float64(rs.MinShards)
 }
 
-// SlabSizeNoRedundancy returns the size of a slab without added redundancy.
-func (rs RedundancySettings) SlabSizeNoRedundancy() uint64 {
+// SlabSize returns the size of a slab.
+func (rs RedundancySettings) SlabSize() uint64 {
```

Comment:
Strictly speaking the size with redundancy would be the `SlabSize` since a `Slab` refers to erasure coded data. 
Not sure we have a good term for a `slab` pre-encoding though.